docs/wallet/backups.md

```diff
@@ -0,0 +1,45 @@
+---
+id: backups
+title: Wallet Backups
+---
+
+Currently, the primary to back up the Bitcoin-S wallet now is through creating a `zip`
+archive of the `.bitcoin-s` data folder. This is chosen because to have a complete backup
+for open DLCs you need to restore all the data given from your counter-party as well as some
+generated by your own wallet. Zipping the folder is the easiest way to back up and restore
+this data as it is not something that can be written down on a piece of paper.
+
+## Backing up from the GUI
+
+To create a backup from the GUI simply click the `File` menu, then select `Save Backup`.
+This will bring up a dialog to choose the save location for where to save the file to.
+
```

Comment:
nit: maybe add image here?

Reply:
Not sure how to do that with our docs

Reply:
Take a look at #3299

Reply:
Specifically here: https://github.com/bitcoin-s/bitcoin-s/pull/3299/files#diff-31bcba2ccafa41d46fbbd6d1219f7f1e3b1fb3cad9faa8e4dc521bbb579dd7b3R38

Reply:
nit: maybe we should add a `docs/images/` directory too

Reply:
Added, also fixed the image of the one you added as it was not being displayed